pkg/api/latest/latest.go

```diff
@@ -33,6 +33,7 @@ var (
 	// RegisterGroup is a shortcut to allGroups.RegisterGroup.
 	RegisterGroup = allGroups.RegisterGroup
 	// GroupOrDie is a shortcut to allGroups.GroupOrDie.
+	// TODO: We will abandon this function and checking the error returned by the Group() function.
```

Comment:
Rather, change to say this should only be called if it's impossible for there to be an error.